### DISCUSSION TITLE
Some routes give 403 error

### DISCUSSION THREAD

**Discussion by mateferrox** [spatie/laravel-permission#2079] [Q&A]
Hello,
I have installed your package, and currently I have a problem with some routes located in the web.php file (for example: "preventivi"):
**web.php*
![1](https://user-images.githubusercontent.com/63313649/166097448-27c806b7-de0d-42ab-b570-6024ccb12b0c.PNG)
*
If I go to the first route ("preventivi") it returns me the error:
403 | User does not have the right permissions. 
it's strange, because in the layout blade file I set it like this:
**backend.blade.php**
![2](https://user-images.githubusercontent.com/63313649/166097160-7e2c99b0-ebc5-4ebf-8621-9fd59fb4b209.PNG)
According to your documentation, I have added in the kernel.php file and in Model/User.php:
**kernel.php**
`'permission' => \Spatie\Permission\Middlewares\PermissionMiddleware::class,`
**User.php**
`use HasFactory, Notifiable, SoftDeletes, **HasRoles**;`
I created the permission and the role through a seeder, always following the documentation:
` $pm = Role::create(['name' => 'pm']);`
`$pm->givePermissionTo('gestione preventivi');`
`$user3 = User::where(['id' => 3])->first();`
`$user3->assignRole('pm');`
A db has the permission with its role, and I also cleared the cache.
Would you kindly know how to give me a hand?

**Comment by erikn69** [comment]
The guard_name??

**Reply by mateferrox** [reply]
I using web

**Reply by erikn69** [reply]
I saying that check the guard name on db
